projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -202,14 +202,14 @@ export class TuiInputDateRangeComponent
     }
 
     get computedValue(): string {
-        const {value, nativeValue, activePeriod} = this;
```

Comment:
in some cases (like that) we creating extra consts, that can be unused below, it's ok?

Reply:
fixed